skypy/galaxies/tests/test_vdf_sampler.py

```diff
@@ -0,0 +1,7 @@
+from skypy.galaxies import vdf_sampler
+import numpy as np
+
+
+def test_vdf_sampler():
```

Comment:
Add some more tests. Also use the test functions of the examples in the main comment as an example.

Reply:
Done.